pydantic_ai_slim/pydantic_ai/_tool_manager.py

```diff
@@ -57,6 +57,16 @@ def tool_defs(self) -> list[ToolDefinition]:
 
         return [tool.tool_def for tool in self.tools.values()]
 
+    def should_parallel_tool_call(self, calls: list[ToolCallPart]) -> bool:
+        """Whether to allow parallel tool calls for a list of tool calls."""
+        for call in calls:
+            if not (tool_def := self.get_tool_def(call.tool_name)):
```

Comment:
This means if nonexistent tools are called, we'll never run in parallel -- I don't think that's necessary